gui/src/main/java/io/bisq/gui/util/GUIUtil.java

```diff
@@ -240,6 +242,38 @@ public CurrencyListItem fromString(String s) {
         };
     }
 
+    public static StringConverter<TradeCurrency> getTradeCurrencyConverter(
+            String postFixSingle,
+            String postFixMulti,
+            Map<String, Integer> offerCounts) {
+        return new StringConverter<TradeCurrency>() {
+            @Override
+            public String toString(TradeCurrency tradeCurrency) {
+                String code = tradeCurrency.getCode();
+                Optional<Integer> offerCountOptional = Optional.ofNullable(offerCounts.get(code));
+                final String displayString;
```

Comment:
Apologies for being daft, but I don't know what you mean by "extracting the code from the toString method of the getTradeCurrencyConverter() method below".  

The  getTradeCurrencyConverter() method below is @Deprecated public static StringConverter<TradeCurrency> getTradeCurrencyConverter() ,
and I intend to delete it in the next commit because nothing calls it.   

Or,  your suggestion has nothing to do with the @Deprecated converter method.  

Can you help me understand what repetition I need to fix?

Reply:
@ghubstan I missed your @Deprecated annotation below. If the method is not used anymore after your changes, you can also just delete it. I thought it is still used at some place.

Reply:
So actually everything was fine with your PR from the beginning 😬. Sorry about the confusion and thanks for your contribution 👍